clusterloader2/pkg/test/simple_test_executor.go

```diff
@@ -86,8 +86,19 @@ func (ste *simpleTestExecutor) ExecuteTest(ctx Context, conf *api.Config) *error
 		if ctx.GetClusterLoaderConfig().ReportDir == "" {
 			klog.Infof("%v: %v", summary.SummaryName(), summary.SummaryContent())
 		} else {
+			testDinstinctor := ""
+			if ctx.GetClusterLoaderConfig().TestScenario.Identifier != "" {
+				testDinstinctor = "_" + ctx.GetClusterLoaderConfig().TestScenario.Identifier
+			}
 			// TODO(krzysied): Remember to keep original filename style for backward compatibility.
-			filePath := path.Join(ctx.GetClusterLoaderConfig().ReportDir, summary.SummaryName()+"_"+conf.Name+"_"+summary.SummaryTime().Format(time.RFC3339)+"."+summary.SummaryExt())
+			filePath := path.Join(ctx.GetClusterLoaderConfig().ReportDir, summary.SummaryName()+
+				"_"+
```

Comment:
```
fileName := strings.Join([]string{summary.SummaryName(), conf.Name+testDistinctor, summary.SummaryTime.Format(...), "_")
filePath := path.Join(reportDir, strings.Join([]string{fileName, extension}, ".")
```

will be a bit more readable what is happening here

Reply:
Done.